.readthedocs.yml

```diff
@@ -10,12 +10,10 @@ build:
   os: ubuntu-22.04
   tools:
     # Use most recent version supported by Numba
-    python: '3.10'
+    python: '3.11'
   apt_packages:
   - graphviz
   jobs:
-    pre_create_environment:
-    - python -m pip install --upgrade setuptools pip
```

Comment:
I don't know if this is necessary or not.

Reply:
I suppose the answer is no, since you removed it and all the tests are still passing?

Reply:
That does appear to be the case!